opencensus/common/monitored_resource/monitored_resource.py

```diff
@@ -54,8 +42,8 @@ def get_instance():
 
     Supported environments include:
 
-    1. 'gke_container'
-    - https://cloud.google.com/monitoring/api/resources#tag_gke_container
+    1. 'k8s_container'
+    - https://cloud.google.com/monitoring/api/resources#tag_k8s_container
```

Comment:
Those URLs (including the ones below) shouldn't point to Stackdriver documentation, since this is our core library (not exporters). Please update this to point to the [specs](https://github.com/census-instrumentation/opencensus-specs/tree/master/resource).

Reply:
I dropped the whole comment since we link to the resource spec from the `Resource` docstring.